### ISSUE TITLE
Field Trip - (27502)

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#2319]
**In Progress Reports**

> Player doesn't actually hide in the crate

**Comment by Kestrana** [comment]
Yes the player is supposedly hiding in the crate during the RP sequence. Not sure if this can be done (or just move the player behind the crate/enlarge the crate?). Cosmetic issue doesnt affect completion.
